### PR TITLE
audio player props and styling options

### PR DESCRIPTION
**Pull request by gunjankothari**

### CODE REVIEW

File: packages/core/src/mappings/AudioPlayer.ts [hunk 1]
```diff
@@ -23,5 +43,21 @@ export const SEED_DATA = {
...
+    sliderColor: createColorProp({
+      label: "Thumb Color",
+      defaultValue: "black",
+    }),
+    completedTrackColor: createColorProp({
+      label: "Completed Track Color",
+      defaultValue: "white",
+    }),
+    remainingTrackColor: createColorProp({
+      label: "Remanining Track Color",
```
Comment (bluerssen) [inline packages/core/src/mappings/AudioPlayer.ts:55]: typo

Comment (gunjankothari) [reply packages/core/src/mappings/AudioPlayer.ts]: fixed

**Comment by bluerssen** [comment]
In Draftbit, the default styles set are generating red screens
Screen Shot 2022-11-17 at 8 23 34 AM

**Review by bluerssen — CHANGES_REQUESTED** [review]
See other comment

**Review by bluerssen — APPROVED** [review]

**Pull request merged**